.buildkite/engineer

```diff
@@ -13,7 +13,7 @@ fi
 if ! type "engineer" > /dev/null; then
     # Setup Prisma engine build & test tool (engineer).
     set -e
-    curl --fail -sSL "https://prisma-engineer.s3-eu-west-1.amazonaws.com/1.23/latest/$OS/engineer.gz" --output engineer.gz
+    curl --fail -sSL "https://prisma-engineer.s3-eu-west-1.amazonaws.com/feat/mongo-5-pipeline/b8e67622c382423ef194e0199f2db5d7921a7860/$OS/engineer.gz" --output engineer.gz
```

Comment:
This may be intentional, but here goes anyways: I usually work with `/latest` to always grab the latest release of the engineer branch I'm targeting. Makes upgrading super easy, because you can actually re-run buildkite steps and builds as it fetches the latest on each run, which doesn't require you to build everything all the time / make commits all the time to test.